### DISCUSSION TITLE
We can have a Nepali to English and English to Nepal converter online service

### DISCUSSION THREAD

**Discussion by gorkhadev** [TourismKit/beautifulnepal.org#1] [Ideas]
A website where one can specially tourist will be able to quickly learn the Nepali language or vice versa i.e Nepali can learn English.

**Comment by devsdenepal** [comment]
## I think we can create a section* for learning Nepali where :
- [x] They'll know basic things by interactive engagement learning
- [x] Then they'll cover the most important content in Nepali language
- [x] We should also have a revision part or exercise to implement what they learnt.


---
 /* = subject to change.
Here "they" refers to client and "we" refers to developers.
